docs/design-docs/00-template.md

```diff
@@ -0,0 +1,90 @@
+# Title
+
+* Author(s): \<your name\>, \<your github alias\>
+* Approver: \<kpt-maintainer\>
+
+>    Every feature will need design sign off an PR approval from a core
+>    maintainer.  If you have not got in touch with anyone yet, you can leave
+>    this blank and we will try to line someone up for you.
+
+## Why
+
+Please provide a reason to have this feature.  For best results a feature should
+be addressing a problem that is described in a github issue.  Link the issues
+in this section.  The more user requests are linked here the more likely this
+design is going to get prioritized on the roadmap.
+
+It's good to include some background about the problem, but do not use that as a
+substitute for real user feedback.
+
+## Design
+
+Please describe your solution. Please list any:
+
+* new config changes
+* interface changes
+* design assumptions
+
+For a new config change, please mention:
+
+* Is it backwards compatible? If not, what is the migration and deprecation 
+  plan?
+
+
+## User Guide
```

Comment:
Suggest: 

`## Design Overview` --> `## User Guide` (or `## User Stories`) --> `## Design Details`

Reply:
updated the title.  Not sure if we need to sections for design though.

Reply:
I think asking people to write too much might diminish the participation.